docs/reference/indices/create-index.asciidoc

```diff
@@ -122,3 +122,47 @@ curl -XPUT localhost:9200/test -d '{
 --------------------------------------------------
 
 <1> `creation_date` is set using epoch time in milliseconds.
+
+[float]
+[[index-creation-wait-for-active-shards]]
+=== Wait For Active Shards
+
+By default, index creation will only return a response to the client when the primary copies of 
+each shard have been started, or the request times out. The index creation response will indicate
+what happened:
+
+[source,js]
+--------------------------------------------------
+{
+    "acknowledged": true,
+    "shards_acknowledged": true
+}
+--------------------------------------------------
+
+`acknowledged` indicates whether the index was successfully created in the cluster, while 
+`shards_acknowledged` indices whether the requisite number of shard copies were started for
+each shard in the index before timing out. Note that it is still possible for either 
+`acknowledged` or `shards_acknowledged` to be `false`, but the index creation was successful. 
```

Comment:
An example might be useful here. If `shards_acknowledged` returns false then the primaries aren't ready. If `acknowledged` is false then we haven't made the change to the cluster state but we probably will in the future, right?

I wonder if we shouldn't return 200 in that case. Because the index isn't ready. There is an HTTP code for that, I think, 202 ACCEPTED. Maybe this all should be moved into another issue.